webpack.config.js

```diff
@@ -1,14 +1,43 @@
-{
-  module.exports = {
-    entry: "./lib/index.js",
-    output: {
-      path: __dirname,
-      filename: "./dist/react-dropzone.js",
-      libraryTarget: "var",
-      library: "Dropzone"
+var webpack = require('webpack');
+var path = require('path');
+
+module.exports = {
+	entry: "./src/index.js",
+    devtool: "source-map",
+	output: {
+		path: __dirname,
+		filename: "./dist/react-dropzone.js",
+		libraryTarget: "umd",
+		library: "Dropzone"
+	},
+    module: {
+        loaders: [
+            {
+                include: [
+                    path.resolve(__dirname,"src"),
+                ],
+                test: /\.js$/,
+                loader: 'babel-loader',
+                query: {
+                    presets: ['es2015','react','stage-0'],
```

Comment:
Yes, when you use same babel config in different places, it's better to use `beblrc` or `package.json` files. If you want override it, this is how you do it. See this for example: https://github.com/mxstbr/react-boilerplate/blob/v3.0.0/webpack/webpack.dev.babel.js#L44-L47